### PR TITLE
Module updates and tweaks

### PR DESCRIPTION
**Pull request by josephco**
This should be rebased and merged after #55 and #56

### CODE REVIEW

File: src/content_panel/panel.macros.html [hunk 1]
```diff
@@ -8,7 +8,7 @@
 {% macro content_panel(heading, image_path, image_description=null, image_position='left', class=null, id=null) %}
 <div class="content-panel{% if class %} {{ class }}{% endif %}"{% if id %} id="{{ id }}"{% endif %}>
   <figure class="content-panel-image content-panel-image--{{ image_position }}">
-    <img src="https://casper.imgix.net/{{ image_path }}" alt="{{ image_description }}">
+    <img class="imgix-fluid" src="https://casper.imgix.net/{{ image_path }}" alt="{{ image_description }}">
```
Comment (misscs) [inline src/content_panel/panel.macros.html:11]: You may need to pass in a specific class to target these via imgix.

File: src/hero/_base.scss [hunk 2]
```diff
@@ -7,6 +7,7 @@
 .hero {
   display: flex;
   flex-direction: column;
+  align-items: center;
   justify-content: center;
   min-height: 300px;
```
Comment (misscs) [inline src/hero/_base.scss]: It's going to be better for us if we avoid pixel values whenever possible. When we want to set height, let's think about it in terms of viewport units: what is the proportion/ratio of the screen we want the element to take up?

**Comment by misscs** [comment]
Not sure if this is ready for review or if you want intermediate feedback. Went ahead and offered my two cents. I'm DTM pending height fix.

File: src/hero/_base.scss (hunk 2 above)
Comment (josephco) [inline src/hero/_base.scss]: I couldn't find a nice viewport unit, so I'm just going to use a modular scale value for now until I can work with design a bit more on that.

**Comment by josephco** [comment]
@misscs I've replaced all the min-height pixel values with modular scale values that are roughly equivalent. Will merge this once we are cool with footer and backgrounds.

**Pull request merged**